Calibration/LumiAlCaRecoProducers/plugins/CorrPCCProducer.cc

```diff
@@ -580,8 +580,6 @@ void CorrPCCProducer::dqmEndRunProduce(edm::Run const& runSeg, const edm::EventS
       throw std::runtime_error("PoolDBService required.");
     }
 
-    delete pccCorrections;
```

Comment:
This deletion can (must) be avoided here because writeOne is used in its ["deprecated" form](https://github.com/cms-sw/cmssw/blob/0eada5d56f1c0867840f0281cc17d53f640c46cb/CondCore/DBOutputService/interface/PoolDBOutputService.h#L109): correct?
Shouldn't you try to move to the "non deprecated" form with the reference as argument, instead of the pointer?

Reply:
@perrotta The idea was to minimise the changes in this PR, and leave to the various code author the migration to the new interface. Here we clearly had to options: 1. Remove the delete and leave the old interface 2. Leave the delete and move to new (reference-based) interface.  I think 2 would not be satisfactory since we are left with the use of pointers that has no justification. So, we should also remove the use of pointers.